.travis.yml

```diff
@@ -3,21 +3,57 @@ matrix:
   include:
     - otp_release: 18.3
       elixir: 1.4
+      env:
+       - COWBOY_VERSION=1.0 ELIXIR_ASSERT_TIMEOUT=2000
```

Comment:
If anyone knows a better way of doing this, I'm all ears. We don't want the cowboy 2 tests running on OTP 18 as OTP 19 is required to compile cowlib.